starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -66,4 +67,47 @@ public void create() {
     Assert.assertTrue(inventoryPostInsertion.size() == existingInventory.size() + 1);
     Assert.assertTrue(createdInventory.getId().equals(returnedInventory.getId()));
   }
+  
+  /**
+   * Test Delete method with valid id.
+   */
+  @Test
+  public void delete() {
+    // Put an inventory in the databse so there's something to delete
+    Inventory inventory = new Inventory();
+    inventory.setName(NAME);
```

Comment:
Besides the name and productType, what is the other attribute that all objects should have, and should have initialized (even if the value is going to change)?

Reply:
Added `inventory.setId(null);` to both delete() test methods.

Reply:
I realize this was not necessary - see long comment below.

Reply:
Okydoky no sweat. I removed the lines (: